python/ray/dataframe/test/test_dataframe.py

```diff
@@ -883,18 +884,27 @@ def test_nlargest():
         ray_df.nlargest(None, None)
 
 
-def test_notna():
-    ray_df = create_test_dataframe()
-
-    with pytest.raises(NotImplementedError):
-        ray_df.notna()
-
+@pytest.fixture
+def test_notna(ray_df, pandas_df):
+    pandas_df_nulls = pd.DataFrame({'col1': [np.NaN, np.NaN, np.NaN, np.NaN],
```

Comment:
This test dataframe looks really useful! Could you instead move it above to its own test so all the test methods could run through it? Move to `test_nan_dataframe()`.